libs/@local/hash-isomorphic-utils/src/flows/example-flow-definitions.ts

```diff
@@ -6,77 +6,6 @@ import type {
 } from "./action-definitions";
 import type { FlowDefinition } from "./types";
 
-export const researchTaskFlowDefinition: FlowDefinition = {
```

Comment:
Can we keep this? Or maybe export a `researchTaskFlowDefinition` and `researchTaskWithAnswerFlowDefinition` based on calling the new function? I was using it to build the DAG UI (view for a non-executed Flow)

Reply:
Okay, sure have brought it back in https://github.com/hashintel/hash/pull/4305/commits/5648d19c37ba241dbfdaaf7f3f3f564950065aea